xDrip/Scenes/Settings/SettingsChartRanges/SettingsChartRangesPresenter.swift

```diff
@@ -19,10 +19,195 @@ protocol SettingsChartRangesPresentationLogic {
 final class SettingsChartRangesPresenter: SettingsChartRangesPresentationLogic {
     weak var viewController: SettingsChartRangesDisplayLogic?
     
+    private lazy var settings: Settings = {
+        return User.current.settings ?? Settings()
+    }()
+    
     // MARK: Do something
     
     func presentLoad(response: SettingsChartRanges.Load.Response) {
-        let viewModel = SettingsChartRanges.Load.ViewModel()
+        let tableViewModel = BaseSettings.ViewModel(
+            sections: [
+                createNormalSection(response: response),
+                createAboveNormalSection(),
+                createUrgentSection(response: response)
+            ]
+        )
+        
+        let viewModel = SettingsChartRanges.Load.ViewModel(tableViewModel: tableViewModel)
         viewController?.displayLoad(viewModel: viewModel)
     }
+    
+    private func createNormalSection(response: SettingsChartRanges.Load.Response) -> BaseSettings.Section {
+        var highValue = settings.warningLevelValue(for: .high)
+        var lowValue = settings.warningLevelValue(for: .low)
+        highValue = GlucoseUnit.convertFromDefault(highValue)
+        lowValue = GlucoseUnit.convertFromDefault(lowValue)
+        
+        let cells: [BaseSettings.Cell] = [
+            createPickerCell(
+                .notHigherLess,
+                detailValues: [highValue, lowValue],
+                pickerValueChanged: response.pickerValueChanged
+            )
+        ]
+        
+        let attrHeader = NSMutableAttributedString()
+        
+        let textAttachment = NSTextAttachment()
+        textAttachment.image = UIImage(named: "greenOval")
```

Comment:
Wouldn't it be more convenient to not use `AttributtedString` for this, but pass a custom `UIView` instead?